src/components/RoleSelectOnboardOne/RoleSelectOnboardOneStyles.tsx

```diff
@@ -1,80 +1,88 @@
-import styled from "styled-components"
+import styled from 'styled-components'
 
 const maxWidthMedia = '920px'
 
 export const TopContainerStyles = styled.div`
-  background: linear-gradient(180deg, #FFF5F1 0%, #FEEAE3 45.27%, #FFDBCC 94.31%);
+  background: linear-gradient(
+    180deg,
+    #fff5f1 0%,
+    #feeae3 45.27%,
+    #ffdbcc 94.31%
+  );
   width: 100%;
   min-height: calc(318px + 164px + 59px);
   @media (max-width: ${maxWidthMedia}) {
     min-height: 444px;
   }
 
   @media (max-width: 420px) {
-    min-height: 544px;
+    min-height: 5440px;
   }
 
   @media (max-width: 360px) {
     min-height: 644px;
   }
 `
-
 export const MainContainer = styled.div`
-  position: absolute;
-  transform: translateX(-50%);
-  left: 50%;
-  width: 90%;
-  padding: 47px 14px 14px 16px;
-  display: flex;
-  justify-content: center;
-  @media (max-width: 920px) {
+  max-width: 95%;
+  margin: auto;
+  @media (min-width: 600px) {
+    max-width: 80%;
+    margin: auto;
+  }
+  @media (max-width: 850px) {
     & > div:nth-child(1) {
       display: none;
     }
-
+  }
+  @media (min-width: 850px) {
```

Comment:
please implement query helper as mentioned in https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300